pkg/kubelet/prober/worker.go

```diff
@@ -230,7 +238,7 @@ func (w *worker) doProbe() (keepGoing bool) {
 	if c.Started != nil && *c.Started {
 		// Stop probing for startup once container has started.
```

Comment:
was this comment incorrect before?

Reply:
I am confused as comment hasn't changed while return value is opposite to what it was before =)

Reply:
Yes, I cannot believe I let it through 4 or 5 releases...
Of course it's harmless to let the worker continue, but we should be lean and avoid wasting resources.